From 6b857404551a0e03de1f729c378a8e6099831ae0 Mon Sep 17 00:00:00 2001
From: Michael Dickens <michael.dickens@ettus.com>
Date: Mon, 31 Aug 2015 09:44:56 -0400
Subject: [PATCH 01/13] b200: on startup when the bad USB state is detected,
 print this out for the user to see, with brevity.

---
 host/lib/usrp/b200/b200_impl.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/host/lib/usrp/b200/b200_impl.cpp b/host/lib/usrp/b200/b200_impl.cpp
index 6bd49e0..2a590b7 100644
--- a/host/lib/usrp/b200/b200_impl.cpp
+++ b/host/lib/usrp/b200/b200_impl.cpp
@@ -259,6 +259,7 @@ static device::sptr b200_make(const device_addr_t &device_addr)
         return device::sptr(new b200_impl(device_addr, handle));
     }
     catch (const uhd::usb_error &e) {
+        UHD_MSG(status) << "Detected bad USB state; resetting." << std::flush;
         libusb::device_handle::sptr dev_handle(libusb::device_handle::get_cached_handle(
             boost::static_pointer_cast<libusb::special_handle>(handle)->get_device()
         ));
-- 
2.1.4

